migration/sql-files/012-add-email-verified.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE users ADD COLUMN email_verified BOOLEAN DEFAULT false;
+UPDATE users SET email_verified = true;
```

Comment:
+1 I think it's OK if we just set all existing emails to true for now. IMO we can afford skip Auth-Keycloak migration at this point.

Reply:
yeah. that would be a fair assumption to make for now.
>we can afford skip Auth-Keycloak migration at this point.

Do you mean, the part where we need to refresh the token?

Reply:
No, no. We have to do the token refresh :)